miio/integrations/humidifier/zhimi/airhumidifier.py

```diff
@@ -99,6 +100,7 @@ def mode(self) -> OperationMode:
         return OperationMode(self.data["mode"])
 
     @property
+    @sensor("Temperature", unit="°C", device_class="temperature")
```

Comment:
```suggestion
    @sensor("Temperature", unit="C", device_class="temperature")
```
I'm starting having second thoughts on how to define the unit information, and I'm currently wondering if it should be an enum to leave the conversion to string to the downstream users... I have no clear answer for this right now, but we should revisit this at some point later on.

Reply:
home-assistant use °C for TEMP_CELSIUS. https://github.com/home-assistant/core/blob/a641bbc352b7559c9491971fa6fc06437f4ab964/homeassistant/const.py#L507 

Will we have some step to convert from C to TEMP_CELSIUS?

Reply:
I think we may need to create some sort of mapping. I'm wondering if there are only so few units in the end that we could simply have an enum inside python-miio to expose those, otherwise there are [libraries like pint](https://pint.readthedocs.io/en/stable/) that could allow doing this in a more standardized manner.

In any case, we don't want to directly tie this library to the internal presentations as there are no guarantees that their API (which is an internal one nonetheless) will remain stable in the future.